examples/js/src/main/kotlin/com/js/sample/Handler.kt

```diff
@@ -0,0 +1,20 @@
+package com.js.sample
+
+import com.kotlin.aws.js.runtime.client.LambdaHTTPClient
+import com.kotlin.aws.js.runtime.objects.LambdaContext
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+
+class Handler {
+    suspend fun handler(context: LambdaContext, request: String) {
+        LambdaHTTPClient.invoke(
```

Comment:
Why do you need to use external LambdaHTTPClient for the example? LambdaHTTPClient should be internal class in the runtime module. Let's use plain "Hello world" example